pkg/types/ip.go

```diff
@@ -38,17 +38,19 @@ func Addr2Ints(any string) (int, int64, int64, int64, int64, error) {
 			return -1, 0, 0, 0, 0, errors.Wrapf(err, "while parsing range %s", any)
 		}
 		return Range2Ints(*net)
-	} else {
-		ip := net.ParseIP(any)
-		if ip == nil {
-			return -1, 0, 0, 0, 0, fmt.Errorf("invalid address")
-		}
-		sz, start, end, err := IP2Ints(ip)
-		if err != nil {
-			return -1, 0, 0, 0, 0, errors.Wrapf(err, "while parsing ip %s", any)
-		}
-		return sz, start, end, start, end, nil
 	}
+
```

Comment:
not 100% on you with this one for comprehension but I won't push :)

Reply:
I just liberated the else there, changed nothing more